tests/Unit/DataStructures/Test_Matrix.py

```diff
@@ -21,24 +22,30 @@ def test_getitem(self):
         self.assertEqual(M[2, 1], 1)
 
     def test_string(self):
-        M = Matrix(2,2)
-        M[0,0] = 1
-        M[0,1] = 2
-        M[1,0] = 3
-        M[1,1] = 4
+        M = Matrix(2, 2)
+        M[0, 0] = 1
+        M[0, 1] = 2
+        M[1, 0] = 3
+        M[1, 1] = 4
         self.assertEqual(str(M),
-            '(            1            2 )\n(            3            4 )\n')
-
+        '(            1            2 )\n(            3            4 )\n')
 
     def test_bounds_check(self):
         a = Matrix(2, 2)
-        self.assertRaises(RuntimeError, lambda: a[5,2])
+        self.assertRaises(RuntimeError, lambda: a[5, 2])
 
         def assignment_test():
-            a[5,2] = 8
+            a[5, 2] = 8
 
         self.assertRaises(RuntimeError, assignment_test)
 
+    def test_to_numpy(self):
+        M = Matrix(2, 2)
+        M[0, 0] = 1.0
```

Comment:
Add an off-diagonal entry to make sure the data layout is correct

Reply:
probably safest to set all elements to values like 1.4, 4.3, 7.12, -9.35 (they aren't multiples of each other) and test that.